adhocracy4/modules/admin.py

```diff
@@ -26,6 +27,11 @@ class ModuleAdmin(admin.ModelAdmin):
         phase_admin.PhaseInline
     ]
     list_filter = ('project__organisation', 'project')
+    if hasattr(settings, 'A4_BLUEPRINT_TYPES'):
+        list_filter += ('blueprint_type',)
+    else:
+        exclude = ['blueprint_type']
```

Comment:
I just realized, that maybe we dont want to show the type at all in admin? Or make it not editable? So modules cant be assigned a wrong type.. What do you think @fuzzylogic2000?

Reply:
Hmm, if we filter by it, we should also show it.
Yes, and I think, making it not editable is a good idea. If we absolutely need to change it, we can still do that in the shell.